### DISCUSSION TITLE
Error while installing mqtt LXC container...PiMox

### DISCUSSION THREAD

**Discussion by cloubier** [tteck/Proxmox#122] [General]
Hi , i have an error occurs when installing MQTT lxc container :
![image](https://user-images.githubusercontent.com/50082017/161388973-32fe7910-4b97-456b-a804-cb76b0958d8e.png)

I use pimox (proxmox on RPI4)...

Any idea ?

**Comment by tteck** [comment]
These scripts do not support PiMox. Sorry

**Reply by cloubier** [reply]
So sad! Maybe in a future version! :-)
